.github/workflows/docker-build-push.yml

```diff
@@ -52,6 +56,7 @@ jobs:
         uses: docker/bake-action@v1.7.0
         with:
           push: true
```

Comment:
Set platform on Push step. This way images will be build and tested on default platform (`linux/amd64`) and pushed for multiple platforms.

Reply:
done